src/Provider/Provider.Web/Controllers/RecruitmentDashboardController.cs

```diff
@@ -0,0 +1,27 @@
+using System.Threading.Tasks;
+using Esfa.Recruit.Provider.Web.Configuration.Routing;
+using Esfa.Recruit.Provider.Web.Orchestrators;
+using Microsoft.AspNetCore.Hosting;
+using Microsoft.AspNetCore.Mvc;
+
+namespace Esfa.Recruit.Provider.Web.Controllers
+{
+    [Route(RoutePaths.VacanciesRoutePath)]
+    public class RecruitmentDashboardController : Controller
+    {
+        private readonly DashboardOrchestrator _orchestrator;
+        private readonly IHostingEnvironment _hostingEnvironment;
+
+        public RecruitmentDashboardController(DashboardOrchestrator orchestrator, IHostingEnvironment hostingEnvironment)
+        {
+            _orchestrator = orchestrator;
+            _hostingEnvironment = hostingEnvironment;
+        }
+
+        [HttpGet("", Name = RouteNames.RecruitmentDashboard_Get)]
```

Comment:
The route name should be "dashboard" here. And the route name on old dashboard should be changes to "vacancies"